test/verify.test.js

```diff
@@ -33,3 +33,19 @@ test('Throw SemanticReleaseError if "changelogFile" option is a whitespace Strin
   t.is(error.name, 'SemanticReleaseError');
   t.is(error.code, 'EINVALIDCHANGELOGFILE');
 });
+
+test('Throw SemanticReleaseError if "changelogTitle" option is not a String', t => {
+  const changelogTitle = 42;
+  const [error] = t.throws(() => verify({changelogTitle}));
+
+  t.is(error.name, 'SemanticReleaseError');
+  t.is(error.code, 'EINVALIDCHANGELOGTITLE');
+});
+
+test('Throw SemanticReleaseError if "changelogTitle" option is a whitespace String', t => {
+  const changelogTitle = '  \n \r ';
+  const [error] = t.throws(() => verify({changelogTitle}));
+
+  t.is(error.name, 'SemanticReleaseError');
+  t.is(error.code, 'EINVALIDCHANGELOGTITLE');
+});
```

Comment:
Can change the tests:
- `Verify String "changelogFile"` into `Verify String "changelogFile" and "changelogTitle"`
- `Verify undefined "changelogFile"` into `Verify undefined "changelogFile" and "changelogTitle"`

Reply:
done